public/apps/configuration/panels/permission-tree.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { EuiTreeView, EuiIcon } from '@elastic/eui';
+import { Node } from '@elastic/eui/src/components/tree_view/tree_view';
+import React from 'react';
+import { ActionGroupItem, DataObject } from '../types';
+
+const MAX_DEPTH = 5;
```

Comment:
Just want to confirm that the max height with scroll function is not a concern since we set a max depth now.

Reply:
There is no max height given yet. Even we need to limit height, I would prefer wrap this component with a height-limited `<div>` rather than implement it in this component.